knx/dpt/types_registry.go

```diff
@@ -36,14 +30,14 @@ func (r *Registry) add(d DatapointValue) error {
 	name = name[:len(name)-3] + "." + name[len(name)-3:]
 
 	// Register the type
-	r.lut[name] = d
+	(*r)[name] = d
```

Comment:
If this methods operates on `Registry` instead, you don't need to dereference it.

Reply:
Ok. I'm still quite new to golang so I thought it might be bad to pass around larger structures by-value, but I'm willing to learn... :-) Will change as you suggest.

Reply:
`map` is itself a reference already, so it won't be passed by value :)